survive-the-monsters.js

```diff
@@ -1,45 +1,94 @@
 import * as Phaser from "./node_modules/phaser/dist/phaser.esm.js";
 
-const GRAVITY = 200;
+const movementConfig = {
+    speed: 300,
+};
+
+const GRAVITY = 0;
 const SCENE_WIDTH = 1920;
 const SCENE_HEIGHT = 1080;
 const SPRITE_SIZE = 128;
+const ENEMIES_COUNT = 2;
 
-const actors = {};
+const actors = {
+    enemies: new Array(ENEMIES_COUNT),
+};
 
-class GameScene extends Phaser.Scene {
-    preload () {
-        this.load.image("player", "sprites/players/mvp_player/idle_0.png");
-        this.load.image("monster_0", "sprites/monsters/mvp_monster_0/idle_0.png");
-        this.load.image("monster_1", "sprites/monsters/mvp_monster_1/idle_0.png");
-    }
-    create () {
-        actors.player = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "player");
-        actors.player.setVelocity(100, 0);
-        actors.player.setBounce(1, 1);
-        actors.player.setCollideWorldBounds(true);
-
-        actors.monster_0 = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "monster_0");
-        actors.monster_0.setVelocity(-150, 10);
-        actors.monster_0.setBounce(1, 1);
-        actors.monster_0.setCollideWorldBounds(true);
-        
-        actors.monster_1 = this.physics.add.image(SPRITE_SIZE, SPRITE_SIZE, "monster_1");
-        actors.monster_1.setVelocity(-200, 0);
-        actors.monster_1.setBounce(1, 1);
-        actors.monster_1.setCollideWorldBounds(true);
+const sceneCenterX = SCENE_WIDTH / 2;
+const sceneCenterY = SCENE_HEIGHT / 2;
+const enemiesX = new Float64Array(ENEMIES_COUNT);
+const enemiesY = new Float64Array(ENEMIES_COUNT);
+const input = [0, 0];
+let playerX = 0;
+let playerY = 0;
+let inputKeys;
```

Comment:
**nitpick**: Initialize `inputKeys` directly with the keyboard keys to avoid a separate declaration.